app-utils/oxipng/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=oxipng
+PKGSEC=utils
+PKGDEP="gcc-runtime"
+BUILDDEP="rustc llvm"
+PKGDES="A multithreaded lossless PNG/APNG compression optimizer"
+USECLANG=1
```

Comment:
Minor nit - consider adding a blank line before this to make defines more readable (separating basic metadata and build options).